src/worker.cc

```diff
@@ -825,7 +845,6 @@ void from_json(rapidjson::Document& doc, Options& options) {
 
   // Parse all of the costing options in their specified order
   sif::ParseCostingOptions(doc, "/costing_options", options);
-  options.set_costing(costing);
```

Comment:
Removed, since its being set [above](https://github.com/valhalla/valhalla/pull/2685/files#diff-dda1eacb6be34d7946468f09844f48890c1266509f634990ec27263271bd6f48R841)